### PR TITLE
Feature add 3 files

### PR DESCRIPTION
**Pull request by somarib**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
...
+- Technologies used
+
+
+
+## Authors
+
+👤 Somarib Emmanuel
+
+- GitHub: [@somarib](https://github.com/somarib/Hellow-world.git)
+
```
Comment (victorokoroji) [inline README.md]: - [ ] You have done well updating your readme 😄, but I think you should improve on your readme by following the[ template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. Please ensure your readme looks like the template provided 👍🏼 .

File: .gitignore [hunk 2]
```diff
@@ -16,3 +16,6 @@ bin-release/
 # Project files, i.e. `.project`, `.actionScriptProperties` and `.flexProperties`
 # should NOT be excluded as they contain compiler settings and other important
 # information for Eclipse / Flash Builder.
+
+#file to exclude
```
Comment (victorokoroji) [inline .gitignore:2]: - [ ] Kindly remember to add node_modules to gitignore which will be generated while setting up linters in your project 😄.

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @somarib, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 

## Required Changes ♻️

- [ ] Your commit message is descriptive. Kudos to that 👍🏼 . It will be much better if you always capitalize the first letter of your commit messages 😄.
 
- [ ] You did well adding the linter workflow 👍🏼 . Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up linters correctly. Currently, your linters are lacking the stylelint and the web hint config files 👍🏼 .

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hi @somarib,
### I'm Levy, your PR reviewer

Awesome work, you've done :sunglasses: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10
## Status: APPROVED :fire: 

## Highlights :muscle: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 
- Correct usage of Github flow :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
  - [OPTIONAL] Kindly make sure to add a good PR title and PR description to the next projects. It makes your repo looks more professional. :wink: 
     - Example of good PR
image

- [OPTIONAL] I think it would be better to name your feature branches meaningful names in future projects like `navigation-bar`, and `mobile-menu`. It will make our repo looks so professional :wink.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**